### PR TITLE
Adding wiki lingua dataset as new branch

### PR DESCRIPTION
**Pull request by katnoria**
Adding the dataset as new branch as advised here: #1470

### CODE REVIEW

File: datasets/wiki_lingua/README.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/wiki_lingua/README.md:143]: Can you give a brief description of the data fields here ? (url, document, section etc.)

File: datasets/wiki_lingua/README.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+| Hindi       |     9,929     |
+| Czech       |     7,200     |
+| Turkish     |     4,503     |
+
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/wiki_lingua/README.md:139]: Can you provide an example from the dataset ? You can get one by doing `load_dataset` and with `dataset["train"][0]` for example

File: datasets/wiki_lingua/README.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+- id
+- it
+- ja
+- ko
+- nl
+- pt
+- th
+- tr
+- vi
+- zh
```
Comment (lhoestq) [inline datasets/wiki_lingua/README.md:22]:
```suggestion
  arabic:
  - ar
  chinese:
  - zh
  czech:
  - cs
  dutch:
  - nl
  english:
  - en
  french:
  - fr
  german:
  - de
  hindi:
  - hi
  indonesian:
  - id
  italian:
  - it
  japanese:
  - ja
  korean:
  - ko
  portuguese:
  - pt
  russian:
  - ru
  spanish:
  - es
  thai:
  - th
  turkish:
  - tr
  vietnamese:
  - vi
```
can you format the languages this way please ?

**Review by lhoestq — COMMENTED** [review]
Nice thank you ! The dataset script looks all good now :) 

I left my final comments about the readme

File: datasets/wiki_lingua/README.md (hunk 2 above)
Comment (katnoria) [reply datasets/wiki_lingua/README.md]: Thanks I have added this to the readme

File: datasets/wiki_lingua/README.md (hunk 1 above)
Comment (katnoria) [reply datasets/wiki_lingua/README.md]: Sure, I have added this to the readme

**Review by lhoestq — APPROVED** [review]
Perfect thank you !

**Pull request merged**